.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
+jobs:
+  test-ruby:
+    parameters:
+      ruby_version:
+        type: string
+    docker:
+      - image: "cimg/ruby:<< parameters.ruby_version >>"
+      - image: cimg/redis:6.2.6
+      - image: memcached:1.6.18
+    working_directory: ~/sdk
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install --jobs=4 --retry=3
+      - run:
+          name: Run tests
+          command: bundle exec rake
+  publish:
+    docker:
+      - image: cimg/ruby:3.2.0
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: bundle install --jobs=4 --retry=3
```

Comment:
Random question: why 4 jobs and 3 retries here?

If I'm understanding correctly more jobs would make this run faster if we've got the resources for it, so are we limited to 4? Or is that just a safe number to go for?

I'm not very familiar with these options for bundler, so I'm legitimately asking so I can understand the functionality a bit better.

Reply:
Using multiple jobs usually speeds up installation in environments with lots of CPU cores. But I just tested it in CircleCI and it doesn't seem to provide a benefit here (only 2 cores), so I removed it.

The `--retries` part is recommended when using multiple jobs because there's a slight chance of a race condition that could cause the installation to fail. So I removed that too if we aren't using multiple jobs.

Reply:
Got it, makes sense. Thanks Nate!